src/base/memory_monitor_cl.cpp

```diff
@@ -0,0 +1,50 @@
+/*
+ *
+ * Copyright 2016 CUBRID Corporation
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+/*
+ * memory_monitor_cl.cpp - Implementation of client functions of memory monitor
+ */
+
+#include "memory_monitor_cl.hpp"
+
+void mmon_print_server_info (MMON_SERVER_INFO &server_info, FILE *outfile_fp)
+{
+  double mem_usage_ratio = 0.0;
+
+  fprintf (outfile_fp, "====================cubrid memmon====================\n");
+  fprintf (outfile_fp, "Server Name: %s\n", server_info.server_name);
+  fprintf (outfile_fp, "Total Memory Usage(KB): %lu\n", MMON_CONVERT_TO_KB_SIZE (server_info.total_mem_usage));
+  fprintf (outfile_fp, "Total Monitoring Meta Usage(KB): %lu\n\n",
```

Comment:
`Total Memory Usage` includes `Total Memory Usage`. However, it looks there is no relationship if it displays this way. How about the following?
```
Total Memory Usage: XX KB (for meta info: X KB)
```

Reply:
I agree. Done.